src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -307,5 +307,12 @@ trait MarathonConf
     noshort = true,
     hidden = true,
     default = Some(MesosHeartbeatMonitor.DEFAULT_HEARTBEAT_FAILURE_THRESHOLD))
+
+  lazy val drainingTime = opt[Long](
+    "draining_time",
```

Comment:
Ah, I haven't read the code closely. If that's the meaning of 0 then that would be ideal for us.

I'm not really qualified to comment on the code itself but the approach in general gets a big 👍 from me.